docs/guides/algorithmiq-tem.ipynb

```diff
@@ -15,134 +15,82 @@
+    "The method consists of constructing a tensor network representing the inverse of the global noise channel affecting the state of the quantum processor, and then applying the map to informationally complete measurement outcomes obtained from the noisy state to obtain unbiased estimators for the observables. \n",
     "\n",
-    "See reference [1] for further details.\n",
+    "As an advantage, TEM leverages informationally complete measurements to give access to a vast set of mitigated expectation values of observables and has optimal sampling overhead on the quantum hardware, as described in Filippov at al. (2023), [arXiv:2307.11740](https://arxiv.org/abs/2307.11740), and  Filippov at al. (2024), [arXiv:2403.13542](https://arxiv.org/abs/2403.13542). The measurement overhead refers to the number of additional measurements required to perform efficient error mitigation, a critical factor in the feasibility of quantum computations. Therefore, TEM has the potential to enable quantum advantage in complex scenarios, such as applications in the fields of quantum chaos, many-body physics, Hubbard dynamics, and small molecule chemistry simulations. \n",
     "\n",
-    "#### References\n",
+    "In a nutshell, the main features and benefits of TEM can be sumerized as:\n",
     "\n",
-    "1. S. Filippov, M. Leahy, M. A. C. Rossi, G. García-Pérez, Scalable tensor-network error mitigation for near-term quantum computing, [arXiv:1111.6950](https://arxiv.org/abs/2307.11740) [quant-ph]"
+    "1.  **Optimal measurement overhead**: TEM is optimal with respect to [theoretical bounds](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.131.210601), meaning that no method can achieve a smaller measurement overhead. In other words, TEM requires the minimum number of additional measurements to perform error mitigation. This in turns means that TEM uses minimal quantum runtime.\n",
+    "2.  **Cost-effectviness**: Since TEM handles noise mitigation entirely in the post-processing stage, there is no need to add extra circuits to the quantum computer, which not only makes the computational cheaper but also dimishes the risk of introducing additional errors due to the imperfections of quantum devices.\n",
+    "3.  **Estimation of multiple observables**: Thanks to informationally-complete measurements, TEM efficiently estimates multiple observables with the same measurement data from the quantum computer.\n",
+    "4.  **Measurement error mitigation**: TEM Qiskit function also includes a proprietary measurement error mitigation method able to significantly reduce readout errors after a short calibration run.\n",
+    "5.  **Accuracy**: TEM significantly improves the accuracy and reliability of digital quantum simulations, making quantum algorithms more precise and dependable.\n"
```

Comment:
```suggestion
    "1.  **Optimal measurement overhead**: TEM is optimal with respect to [theoretical bounds](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.131.210601), meaning that no method can achieve a smaller measurement overhead. In other words, TEM requires the minimum number of additional measurements to perform error mitigation. This in turns means that TEM uses minimal quantum runtime.\n",
    "2.  **Cost-effectiveness**: Since TEM handles noise mitigation entirely in the post-processing stage, there is no need to add extra circuits to the quantum computer, which not only makes the computational cheaper but also diminishes the risk of introducing additional errors due to the imperfections of quantum devices.\n",
    "3.  **Estimation of multiple observables**: Thanks to informationally-complete measurements, TEM efficiently estimates multiple observables with the same measurement data from the quantum computer.\n",
    "4.  **Measurement error mitigation**: The TEM Qiskit Function also includes a proprietary measurement error mitigation method able to significantly reduce readout errors after a short calibration run.\n",
    "5.  **Accuracy**: TEM significantly improves the accuracy and reliability of digital quantum simulations, making quantum algorithms more precise and dependable.\n"
```